doc/guide/build.hpp

```diff
@@ -8,6 +8,18 @@ FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example, 
 on Ubuntu, you can install mlpack with the following command:
 
+@code
+$ sudo apt-get install mlpack-bin
+@endcode
+
+On Fedora or Red Hat:
+
+@code
+$ sudo dnf install mlpack
+@endcode
+
+For installing only header files for development purposes one could use:
```

Comment:
`libmlpack-dev` installs the development libs, mlpack is not a header only library at least not yet.

Reply:
@zoq I got this info from here https://www.mlpack.org/doc/stable/doxygen/cli_quickstart.html .But looks like mlpack packages don't exist on fedora. I did dnf list all on a fedora instance and found that there is no such package that matches mlpack string on Fedora repository.If it exists by alternate name please suggest one and I will change it. Else i will remove this line as mlpack may not exist on fedora/red hat repositories

Reply:
I checked on Red Hat 8.3.1-5 dont know if earlier versions support.

Reply:
Yes got it. Earlier i was noot seeing the libmlpack.so file will add the info here.